### PR TITLE
Improve OpenGraph meta tags #697

### LINKED ISSUE #697: Improve OpenGraph meta tags
Open graph meta tags need an update/standardization. Sharing incidentdatabase.ai links could do better: 

Facebook:

image

Linked in:

image

Shares could show the incident title, a description, a picture, etc.

### PR DESCRIPTION
**Pull request by pdcp1**
This PR:
- Improves Twitter OpenGraph meta tags
- Improves Facebook OpenGraph meta tags
- Improves Linkedin OpenGraph meta tags

Link to test: https://feature-improve-opengraph--pablo-staging-aiid.netlify.app/

**On a Facebook post:**
image

**On Twitter:**
image

**On Linkedin:**
image

This PR closes #697

### CODE REVIEW

File: site/gatsby-site/src/templates/cite.js [hunk 1]
```diff
@@ -73,6 +73,8 @@ function CitePage(props) {
 
   const sortedReports = sortIncidentsByDatePublished(incidentReports);
 
+  const metaImage = sortedReports.length > 0 ? sortedReports[0].image_url : null;
```
Comment (cesarvarela) [inline site/gatsby-site/src/templates/cite.js:76]: Is there an incident without reports? This probably shouldn't happen 👀

Comment (pdcp1) [reply site/gatsby-site/src/templates/cite.js]: You're right. I removed the unnecessary conditional.

**Comment by pdcp1** [comment]
@cesarvarela This is ready for review again.

**Review by cesarvarela — APPROVED** [review]

**Comment by cesarvarela** [comment]
Approving with a failing test because it is unrelated and known to be flaky.

**Comment by cesarvarela** [comment]
@pdcp1 can you merge staging here, so it is all green and beautiful? 

Is it worth adding some tests for this?

**Comment by pdcp1** [comment]
> @pdcp1 can you merge staging here, so it is all green and beautiful?
> 
> Is it worth adding some tests for this?

@cesarvarela I merged staging and also added a test, please review.

File: site/gatsby-site/cypress/e2e/cite.cy.js [hunk 2]
```diff
@@ -210,4 +210,23 @@ describe('Cite pages', () => {
       `Olsson, Catherine. (2014-08-14) Incident Number 10. in McGregor, S. (ed.) Artificial Intelligence Incident Database. Responsible AI Collaborative. Retrieved on ${date} from incidentdatabase.ai/cite/10.`
     );
   });
+
+  it('Should have OpenGraph meta tags', () => {
```
Comment (cesarvarela) [inline site/gatsby-site/cypress/e2e/cite.cy.js:214]: You could use `cy.query` to fetch the actual incident title and make sure the proper value is there 🤔

Comment (pdcp1) [reply site/gatsby-site/cypress/e2e/cite.cy.js:217]: @cesarvarela done! Now it uses `cy.query`

**Pull request merged**